formula-android-tests/src/test/java/com/instacart/formula/FragmentAndroidEventTest.kt

```diff
@@ -29,7 +30,7 @@ class FragmentAndroidEventTest {
                         configureActivity = {
                             initialContract = TestLifecycleKey()
                         },
-                        contracts =  {
+                        fragmentStore = FragmentFlowStore.init {
```

Comment:
Should we consider a rename from `FragmentFlowStore` to maybe just `FragmentStore` as well? When I see `Flow`, I always just think Coroutine `Flow` so it can be a little jarring to realize that its not that.

Reply:
Agreed, will do in a follow up.